.github/workflows/upload.yaml

```diff
@@ -76,9 +75,6 @@ jobs:
           - trigger: ${{ inputs.triggerNextfluPrivate }}
             workflow: run-nextflu-private-builds.yaml
             image: ${{ inputs.nextfluPrivateDockerImage }}
-          - trigger: ${{ inputs.triggerNextclade }}
-            workflow: run-nextclade.yaml
-            image: ${{ inputs.nextcladeDockerImage }}
```

Comment:
`inputs.nextcladeDockerImage` could still be passed thru to `run-nextclade.yaml` if you'd like.

Reply:
I prefer to keep the interface simpler...if we need a different image here we can run the separate GHA workflow with that image.